Sources/MapboxCoreNavigation/MapMatchingResult.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+import MapboxNavigationNative
+import CoreLocation
+
+/// Provides information about the status of the enhanced location updates generated by the map matching engine of the Navigation SDK.
+public struct MapMatchingResult {
+    
+    /// The best possible location update, snapped to the route or map matched to the road if possible
+    public var enhancedLocation: FixLocation
```

Comment:
Do we really want to expose `FixLocation` here and not `CLLocation`?

Reply:
Right, didn't think about it. Will remove it